packages/runner-ct/src/app/ReporterContainer.tsx

```diff
@@ -15,7 +13,10 @@ import styles from './RunnerCt.module.scss'
 interface ReporterContainerProps {
   state: State
   eventManager: typeof EventManager
-  config: Cypress.RuntimeConfigOptions
```

Comment:
I am really not sure what the correct type definition is for this. It's certainly not `Cypress.RuntimeConfigOptions`, though, there are some properties that definitely exist that are not included in that type. Or, the type is wrong.